C/services/south/ingest.cpp

```diff
@@ -1076,9 +1080,14 @@ void Ingest::useFilteredData(OUTPUT_HANDLE *outHandle,
 	    Logger::getLogger()->info("%s:%d: Input readingSet modified by filter: ingest->m_data=%p, readingSet->getAllReadingsPtr()=%p", 
```

Comment:
Should this be a _debug_ message instead of an _info_?

Reply:
The code should not be three at all as if was #if 0